src/flows/recommendation_api/user_impressions_flow.py

```diff
@@ -63,11 +62,11 @@ def load_feature_group(df: pd.DataFrame, feature_group_name):
 with Flow(FLOW_NAME, schedule=schedule) as flow:
     feature_group = Parameter("feature group", default=f"{config.ENVIRONMENT}-user-impressions-v1")
```

Comment:
```suggestion
    feature_group = Parameter("feature group", default=f"{config.ENVIRONMENT}-user-impressions-v2")
```
We probably need to rename the 'resolved_ids' column to 'corpus_ids'.